src/errors/test-run/index.js

```diff
@@ -118,6 +118,14 @@ export class ActionOptionsTypeError extends ActionError {
     }
 }
 
+export class ActionTextTypeError extends ActionError {
```

Comment:
I suppose this error is specific for the `type` action so maybe we should use it in the error name? `TypeTextArgumentTypeError`, `TypeTextTypeError` or something better.. /cc @inikulin @VasilyStrelyaev